dGame/dPropertyBehaviors/ControlBehaviorMessages/BehaviorMessageBase.cpp

```diff
@@ -5,20 +5,21 @@
 #include "dCommonVars.h"
 
 BehaviorMessageBase::BehaviorMessageBase(AMFArrayValue* arguments) {
-	behaviorId = GetBehaviorIdFromArgument(arguments);
+	this->behaviorId = GetBehaviorIdFromArgument(arguments);
```

Comment:
See update for why I will not be changing this. It required the editing of 20ish files, and that was indiscriminate through a ctrl+f find and replace operation, so it changed quite a few unrelated things.

Reply:
ok but please try to not rope random changes into random prs because you spot them as you go.  it sets a bad precedence of every pr just being effectively "general fixes"